utils/pre-commit

```diff
@@ -11,7 +11,7 @@ then
 	against=HEAD
 else
 	# Initial commit: diff against an empty tree object
-        against=280fc57fade28e35046c3e884e587ffef05d3867
+    against=280fc57fade28e35046c3e884e587ffef05d3867
```

Comment:
This seems unrelated?

Reply:
Fine with me - but the rest (comment, block above) should be made consistent, too, in this case, I think.